pygmt/src/solar.py

```diff
@@ -0,0 +1,92 @@
+"""
+solar - Plot day-night terminators.
+"""
+
+import datetime
+
+import pandas as pd
+from pygmt.clib import Session
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import build_arg_string, fmt_docstring, kwargs_to_strings, use_alias
+
+
+@fmt_docstring
+@use_alias(
+    R="region",
+    J="projection",
+    G="fill",
+    B="frame",
+    W="pen",
+    U="timestamp",
+    V="verbose",
+    X="xshift",
+    Y="yshift",
+    p="perspective",
+)
+@kwargs_to_strings(R="sequence", p="sequence")
+def solar(self, terminator="d", terminator_datetime="", **kwargs):
```

Comment:
Perhaps use `None` as the default argument.
```suggestion
def solar(self, terminator="d", terminator_datetime=None, **kwargs):
```